sparkmagic/sparkmagic/livyclientlib/command.py

```diff
@@ -42,6 +42,19 @@ def execute(self, session):
             response = session.http_client.post_statement(session.id, data)
             statement_id = response[u'id']
             output = self._get_statement_output(session, statement_id)
+        except KeyboardInterrupt as e:
+            msg =  u'Interrupted by user'
+            if statement_id >= 0:
+                response = session.http_client.cancel_statement(session.id, statement_id)
+                if u'msg' not in response or response[u'msg'] != u'canceled':
+                    msg += u', but Livy returns error'
+                else:
+                    session.wait_for_idle()
+            self._spark_events.emit_statement_execution_end_event(session.guid, session.kind, session.id,
+                                                                  self.guid, statement_id, False, e.__class__.__name__,
+                                                                  str(e))
+            # This is ugly, but raising SparkStatementException or returning does not interrupt subsequent cells.
```

Comment:
This is intended. If you want `raising SparkStatementException` to interrupt, you need to set [all_errors_are_fatal](https://github.com/jupyter-incubator/sparkmagic/blob/master/sparkmagic/sparkmagic/utils/configuration.py#L256-L259)

Reply:
I don't necessarily need to raise `SparkStatementException`. My goal is to interrupt the notebook's execution and print a short message to confirm that. I have yet to find a way to achieve this. Any suggestion would be welcome. I will also dig a bit more.

Reply:
Why not create a new Exception to represent this Interruption and make it always fatal?

Reply:
In the case of a fatal exception, the backtrace is pretty long and unhelpful. Is there a way to work around this? Since the user is expecting the notebook to be interrupted, a short message to confirm it is preferred over printing the backtrace.